src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -215,8 +215,6 @@ impl Orchestrator for KubernetesOrchestrator {
 #[derive(Clone, Copy)]
 struct ServiceInfo {
     scale: u16,
-    disk: bool,
-    disk_limit: Option<DiskLimit>,
 }
```

Comment:
(Just pondering) We could remove the `ServiceInfo` struct, but would still need a place to store private information. Maybe the orchestrator could return an opaque handle to store private information, and functions that currently take an ID would use the handle instead?

Reply:
We could do that, but I'm not sure it improves things. We'll remove some state from the orchestrator, but we'll have to create that state in the controller (or at a higher level again). Currently the controller constructs the ID from `(cluster_id, replica_id, generation)`. It would instead need to keep a map relating these IDs to service handles, which would look a lot like the `service_infos` map in the orchestrator now.